CONTRIBUTING.md

```diff
@@ -60,7 +60,10 @@ Your branch should follow this naming convention:
 These commands will help you do this:
 
 ``` bash
+<<<<<<< HEAD
```

Comment:
Looks like some extra hashing is still in here

Reply:
Accepted incoming change.